ckanext/ytp/comments/logic/action/delete.py

```diff
@@ -13,14 +14,14 @@ def comment_delete(context, data_dict):
     user = context['user']
     userobj = model.User.get(user)
 
-    logic.check_access("comment_delete", context, data_dict)
+    check_access("comment_delete", context, data_dict)
 
     # Comment should either be set state=='deleted' if no children,
     # otherwise content should be set to withdrawn text
-    id = logic.get_or_bust(data_dict, 'id')
+    id = get_or_bust(data_dict, 'id')
```

Comment:
is it the preferred way to drop the parent package and only do the functions we use?

Reply:
Either works, but the important thing is that it's using `toolkit` (which is intended as a more stable API) rather than `logic`.

Importing specific functions does have the advantage of minimising changes if we eventually need to import them from somewhere else. If this module had imported the function, instead of importing `logic`, then we wouldn't have needed to touch this line.